### DISCUSSION TITLE
BTT Pad 7 and SOVOL SV06 Plus

### DISCUSSION THREAD

**Discussion by tusculumgolfer** [bassamanator/Sovol-SV06-firmware#135] [Show and tell]
I Klipperized my SV06 Plus last night with the BTT Pad 7, following the guide, and ran into a few hiccups which I thought I would share here to help anyone else doing the same thing. Following the guide, I ended up with everything seeming to run, but moonraker would not connect with Klipper. I used video (use CC for english) https://www.youtube.com/watch?v=zQZPUjVn3Vo and thread (https://github.com/bassamanator/Sovol-SV06-firmware/discussions/16) to finally figure out the default location name for the files in this firmware is "pi" (e.g.klippy_uds_address: /home/pi/printer_data/comms/klippy.sock). To make it work, change "pi" to "biqu" In moonraker.conf in the following line: klippy_uds_address: /home/pi/printer_data/comms/klippy.sock AND cfgs/misc-macros.cfg, change [virtual_sdcard] path: /home/pi/printer_data/gcodes. After this and a restart, it worked!

**Comment by RoisinDubh79** [comment]
If you want to use the included accelerometer, add the following to your printer.cfg:

```
# Documentation https://www.klipper3d.org/Measuring_Resonances.html?h=adxl#configure-adxl345-with-rpi
# Documentation https://www.klipper3d.org/RPi_microcontroller.html

# host MCU service is preinstalled and ready to use with:
[mcu CB1]
serial: /tmp/klipper_host_mcu

[adxl345]
cs_pin: CB1:None
spi_bus: spidev1.1
axes_map: z,y,-x  # Modify according to the actual orientation of the ADXL345 installed on the printer.

[resonance_tester] 
accel_chip:adxl345 
probe_points: 169.5,123.5,20
```


Alternatively, create a file named adxl-direct-pad7.cfg in the cfgs directory and add the information above to it.
Modify osskc.cfg file and add the following line to it:

`# [include ./cfgs/adxl-direct-pad7.cfg]`

Then you can activate/deactivate the accelerometer as needed.
